azure-pipelines.yml

```diff
@@ -159,11 +159,21 @@ jobs:
   steps:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
+    displayName: Bazel Lint (Experimental)
```

Comment:
This looks like it's being applied to the  - template: ci/install-package-dependencies.yml step
This might be a typo?

Reply:
Thanks for catching that! I thought I stashed those lines.